includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -119,7 +119,6 @@ private function filter_attributes( $attributes ) {
 				case 'alt':
 				case 'class':
 				case 'srcset':
-				case 'sizes':
```

Comment:
That sounds good, @delawski. If it's alright, I'll work on this in several hours.

Reply:
@delawski @kienstra Adding a `layout` to an HTML `img` element makes me nervous. I think it would be better actually to introduce an `data-amp-layout` attribute which the sanitizer could convert into a `layout` attribute when converting an `img` into an `amp-img`. This would allow for regular plain HTML in post content to be decorated with the `data-amp-layout` attribute and for the content to be still served as valid HTML. This will also allow for such `img[data-amp-layout]` to be styled with CSS in non-AMP responses. This would play nicely into adding AMP support for Gutenberg blocks as described in https://github.com/Automattic/amp-wp/pull/902#issuecomment-361310795

We'll need to allow `data-amp-layout` attribute to Kses for it to be saved with `unfiltered_html`.

Reply:
Thanks, @westonruter. The 2 commits below convert `'data-amp-layout'` to  `'layout'` in the image sanitizer, and allow `'data-amp-layout'` in `wp_kses()`, when in a `'post'` context.